src/zmq/zmqnotificationinterface.cpp

```diff
@@ -29,6 +33,15 @@ CZMQNotificationInterface::~CZMQNotificationInterface()
     }
 }
 
+void CZMQNotificationInterface::AddJSONNotifiers(UniValue& publish) const
```

Comment:
IMO should not use UniValue here. Return a list instead?

Reply:
I thought about this - returning a list means (because I think we should not return a list of non-const notifiers) that we have to create a new `std::list<const CZMQAbstractNotifier*>` and cannot return the existing `notifiers`.  But on second thought, that seems not too bad - and decoupling from `UniValue` makes sense.  I've changed that accordingly.